sections/semantics-textlevel.include

```diff
@@ -2993,22 +2993,41 @@
       This example shows how the <{bdo}> element can be used to alter directionality formatting:
```

Comment:
I think we should delete this example, because (1) it is redundant given the later example, and (2) it puts too much emphasis on the direction of display. Cut from here....